lib/bundler/definition.rb

```diff
@@ -267,7 +267,7 @@ def lock(file)
       File.open(file, 'wb'){|f| f.puts(contents) }
     rescue Errno::EACCES
       raise Bundler::InstallError,
-        "There was an error while trying to write to Gemfile.lock. It is likely that \n" \
+        "There was an error while trying to write to #{File.basename(file)}. It is likely that \n" \
```

Comment:
Why is this change different from the others? Shouldn't this be `#{Bundler.default_lockfile}`?